CHANGES.rst

```diff
@@ -1006,614 +1006,6 @@ Other changes
   to make it more visible.
 
 
-vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv
-vvv    TO DISTRIBUTE TO PROPER SECTIONS ABOVE    vvv
-vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv
```

Comment:
In an ideal world, those items being removed by 7245b4a0106d59dddaafa194a6b42f4eee83ffba still need to be either distributed to an appropriate section (or if not relevant anymore, deleted). Mainly those are items where I didn't know where to put them (e.g.: not clear between Python compat, pure python, Cython improvement, etc.). It's possible that we might want to add or change sections based on the rest of the lines here.

Reply:
Ah thanks. I'll probably add an "optimisations" section and a "everything else" section.